hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/HoodieTable.java

```diff
@@ -723,11 +736,17 @@ void rollbackInflightInstant(HoodieInstant inflightInstant,
    * @param inflightInstant Inflight Compaction Instant
    */
   public void rollbackInflightLogCompaction(HoodieInstant inflightInstant, Function<String, Option<HoodiePendingRollbackInfo>> getPendingRollbackInstantFunc) {
-    final String commitTime = getPendingRollbackInstantFunc.apply(inflightInstant.requestedTime()).map(entry
-        -> entry.getRollbackInstant().requestedTime())
-        .orElseGet(() -> getMetaClient().createNewInstantTime());
-    scheduleRollback(context, commitTime, inflightInstant, false, config.shouldRollbackUsingMarkers(),
-        false);
+    transactionManager.beginTransaction(Option.empty(), Option.empty());
```

Comment:
can we define an overload for `beginTransaction(empty,empty)` with no args `beginTransaction()` .. Also, why would not we pass in `inflightInstant` for the first arg. I get that the second arg is kind of not useful for every state change.. 

Same q for all other calls. I think the args are just used for logging.. so it helps us trace state changes .